src/contexts/cart-context.tsx

```
@@ -0,0 +1,152 @@
+          }
+        }
+      } else {
+        // Empresa diferente - mostra erro
+        toast.error(`Você só pode adicionar produtos da empresa ${state.enterprise} ao carrinho.`)
+        return state
+      }
+    }
+
+    case "REMOVE_ITEM":
+      const newItems = state.items.filter(item => item.product.id !== action.productId)
+      return {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)